hal/src/sercom/v1/pads.rs

```diff
@@ -125,9 +125,9 @@ macro_rules! pad_alias {
 }
 
 pad_alias!(Sercom0, Sercom1);
-#[cfg(any(feature = "samd21", feature = "min-samd51g"))]
+#[cfg(any(feature = "samd2x", feature = "min-samd51g"))]
 pad_alias!(Sercom2, Sercom3);
-#[cfg(any(feature = "min-samd21g", feature = "min-samd51g"))]
+#[cfg(any(feature = "min-samd2x", feature = "min-samd51g"))]
```

Comment:
What's the difference between `min-samd2x` and `samd2x`? That seems redundant. `min-samd21g` is intended to exclude `samd21e`.